content/guides/rentals-synchronization.md

```diff
@@ -0,0 +1,250 @@
+  scope :by_availabilities, ->(date, length) {
+    unavailable_status = '1'
+    start_point = "DATE_PART('day', TIMESTAMP :date - availability_start_date)::integer+1"
+    availability_to_check_sql = "SUBSTR(availability_map, #{start_point}, :length_of_stay)"
+    where("#{availability_to_check_sql} NOT SIMILAR TO :check_statuses", date: date, length_of_stay: length, check_statuses: unavailable_status)
+  }
+~~~
+
+**SQL Example:**
+
+~~~sql
+SELECT r.id
+FROM rentals r
+WHERE
+  SUBSTRING(r.availability_map,
+            DATE_PART('day', TIMESTAMP '2023-07-01' - r.availability_start_date)::integer + 1,
+            7)
+  NOT SIMILAR TO '1';
+~~~
+
+`2023-07-01` - is a sample start date, `7` is a sample length of stay.
+
+
+## Fetching Rental Prices
+
+Before we start, please read an article [Understanding LOS Records](https://developers.bookingsync.com/guides/understanding-los-records/). It explains what is LOS records and how it works.
+We generate LOS records for the next 18 months, starting from yesterday.
+
+> Attention! `/los-records` is for debugging purposes only! Please don't use it in production mode!
+
+To fetch rental prices, it is recommended to use `/api/ota/v1/los-record-export-urls` endpoint. This endpoint provides CSV files containing LOS records for rentals.
+CSV files look like:
+
+~~~bash
+id;account_id;rental_id;currency;min_occupancy;max_occupancy;kind;day;rates
+45086517195;1;11;EUR;1;4;final_price;2023-07-25;{0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,12501.45,13741.65,14981.85,16222.05,17462.25,18702.45,19942.65,21182.85,22423.05,23663.25,24903.45,26143.65,27383.85,28624.05,29864.25,31104.45,32344.65,33584.85,34825.05,36065.25,37305.45}
+46986331108;2;22;EUR;1;1;final_price;2023-07-25;{0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,4422.6}
+47234184996;3;33;EUR;1;1;final_price;2023-07-25;{0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,4422.6}
+47235754161;4;44;EUR;1;1;final_price;2023-07-25;{0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,4422.6}
+47844857128;5;55;EUR;1;4;final_price;2023-07-25;{0.0,0.0,0.0,0.0,0.0,0.0,1096.2,1252.8,1409.4,1566.0,1722.6,1879.2,2035.8,2192.4,2349.0,2505.6,2662.2,2818.8,2975.4,3132.0,3288.6,3445.2,3601.8,3758.4,3915.0,4071.6,4228.2,4384.8,4541.4,4698.0}
+~~~
+
+There are 3 kinds of LOS records ([LOS kinds](https://developers.bookingsync.com/reference/enums/#los-kinds)):
+
+  1. **rental_price** - Price for the rent only, after all discounts applied.
+  2. **rental_price_before_special_offers** - Price for the rent only, before special offers discounts being applied.
+  3. **final_price** - Price including all required fees and taxes.
+
+If you want to avoid price discrepancy, you have to use **rental_price** and apply all the fees and taxes on your side. Otherwise just use **final_price**.
```

Comment:
May be something like: We highly recommend to use LOS final_price. It may produce a price discrepancy between search_price and booking_price, but we allow that on our side. We expect that booking price won't be lower than in quote request, but it can be higher. 

If you want to avoid price discrepancy, you have to use **rental_price** and apply all the fees and taxes on your side. You need to read https://manual.bookingsync.com/hc/en-us/articles/360005323933-Fees-Services and https://manual.bookingsync.com/hc/en-us/articles/360005323693-Apply-a-new-tax#3

Reply:
You can use something like this:

---

We strongly advise using the LOS `final_price`. Although this may lead to a discrepancy between the `search_price` and the `booking_price`, this variance is permissible within our system. We anticipate that the `booking_price` will not be lower than that stated in the quote request; however, it might be higher.

To avoid any such price discrepancy, you have the option to utilize the `rental_price` and apply all fees and taxes on your end. For an in-depth understanding of handling fees and taxes, please refer to our comprehensive guides on [Fees & Services](https://manual.bookingsync.com/hc/en-us/articles/360005323933-Fees-Services) and [Applying a new tax](https://manual.bookingsync.com/hc/en-us/articles/360005323693-Apply-a-new-tax#3).